resources/views/registerProject.blade.php

```diff
@@ -0,0 +1,218 @@
+@extends('layouts.app')
+
+@section('title', 'Add Project')
+
+@section('content')
+<!-- Internal CSS stylesheet -->
+<style>
+	#uploadImage{
+		margin-top: 1em;
+		margin-bottom: 2em;
+	}
+
+	.hidden{
+		display: none;
+	}
+</style>
+<div class="ui container" style="margin: 4% auto 8%">
+	<div class="ui grid centered">
+		<div class="ten wide column">
+			<div class="ui form error">
+				<!-- Alert Message -->
+				<div id="errorMessage" class="ui error message hidden">
+					<div class="header">Incomplete Information</div>
+					<p>All fields are required for registering a new project.</p>
+				</div>
+				<!-- Project Image -->
+				<div class="projectImg">
+					<img src="https://lorempixel.com/400/400" alt="Project Image" class="ui centered medium circular image"/>
+					<a href="#" class="imgUploader">
+						<button id="uploadImage" class="fluid ui button">Upload Image</button>
```

Comment:
The button style should be consistent with the platform. The style is already defined in the app.scss. 
Buttons should be aligned left when not taking the whole width. Like the example below.

![image](https://user-images.githubusercontent.com/14140121/53279521-cbb44100-36d6-11e9-999b-936faa197a47.png)